tests/features/environment.py

```diff
@@ -44,7 +44,10 @@ def before_all(context):
         'dbname': db_name,
         'dbname_tmp': db_name_full + '_tmp',
         'vi': vi,
+        'pager_boundary': '---boundary---',
     }
+    os.environ['PAGER'] = "{0} {1} {2}".format(
```

Comment:
On lines 15-17, above this part, you still have:

```
 os.environ['LINES'] = "100"
 os.environ['COLUMNS'] = "100"
 os.environ['PAGER'] = 'cat'
```

Since it's not relevant anymore, it should be removed.